sprocs/variants_select_for_assessment_instance_grading.sql

```diff
@@ -7,8 +7,6 @@ CREATE FUNCTION
     ) RETURNS SETOF RECORD
 AS $$
 BEGIN
-    PERFORM assessment_instances_ensure_open(assessment_instance_id);
-
```

Comment:
This `assessment_instances_ensure_open` is no longer valid. We changed `lib/assessment.js#_gradeAssessmentInstance` to close the assessment before invoking this sproc, so the assessment is sometimes closed here. I believe removing this lock is fine. `_gradeAssessmentInstance` is the only place where we use `varaints_select_for_assessment_instance_grading`, so I believe this change won't have any unintended side effects.